packages/editor/src/components/page-attributes/style.scss

```diff
@@ -18,3 +18,99 @@
 		width: 66px;
 	}
 }
+
+.components-parent-page__autocomplete__wrapper {
+	box-sizing: border-box;
+	position: relative;
+}
+
+.components-parent-page__autocomplete__hint,
+.components-parent-page__autocomplete__input {
+	-webkit-appearance: none;
+	border: 2px solid;
+	border-radius: 0; /* Safari 10 on iOS adds implicit border rounding. */
+	box-sizing: border-box;
```

Comment:
Most of these styles were copy pasted from the library right? Most of the attributes are being overwritten by the editor. This could be slimmed down a lot. I would add them one by one as needed.

Reply:
yes, the CSS needs work and generalizing especially when we move to use the component in various places.